### PR TITLE
Botonic-React: change staticAsset function + add unit tests

### PR DESCRIPTION
**Pull request by AlbertGom**
## Description
Changed how `staticAsset` function is searching for the script to resolve the assets path.

## Context
The function is only searching for scripts that end with "webchat.botonic.js" so if the webchat file has another name it won't work.

## Approach taken / Explain the design
Change the query done in the function to check that the script contains "webchat.botonic.js"

## Testing

Added unit tests to test the function.

### CODE REVIEW

File: packages/botonic-react/tests/utils.test.js [hunk 1]
```diff
@@ -26,3 +29,100 @@ describe('Regex serialization / deserialization', () => {
...
+}
+
+describe('staticAsset function', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  test('should return the path if it is a URL', () => {
+    //arrange
+    isURL.mockReturnValue(true)
```
Comment (vanbasten17) [inline packages/botonic-react/tests/utils.test.js:57]: why mocking isURL? in this case it is a url in fact, right?

File: packages/botonic-react/tests/utils.test.js [hunk 2]
```diff
@@ -26,3 +29,100 @@ describe('Regex serialization / deserialization', () => {
...
+    //act
+    const result = staticAsset(url)
+
+    //assert
+    expect(result).toBe(url)
+  })
+
+  test('should resolve the static asset path if it is not a URL and file name is webchat.botonic.js', () => {
+    //arrange
+    isURL.mockReturnValue(false)
```
Comment (vanbasten17) [inline packages/botonic-react/tests/utils.test.js:69]: same here

File: packages/botonic-react/tests/utils.test.js (hunk 1 above)
Comment (AlbertGom) [reply packages/botonic-react/tests/utils.test.js:57]: Mocking it ensures that what I am testing is the behaviour of the `staticAsset` function without relying on the correctness of `isURL` method. In fact, the isURL function should have a unit test itself.

Comment (vanbasten17) [reply packages/botonic-react/tests/utils.test.js:57]: yes, this was exactly what I meant..

**Review by vanbasten17 — APPROVED** [review]

**Review by Iru89 — APPROVED** [review]

**Pull request merged**